.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,6 +4,8 @@ about: Create a report to help us improve
 
 ---
 
+**Note: for support questions, please use our discourse (https://renku.discourse.group/)**.
```

Comment:
Could we use html comments for these headings? Otherwise they often end up in the issue text. Something like 

```
<!-- Note: for support questions, please use our discourse (https://renku.discourse.group/) -->

<!--  Describe the bug --> 
```

Reply:
Done. Anything else?